nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -207,6 +207,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - [xdg-terminal-exec](https://github.com/Vladimir-csp/xdg-terminal-exec), the proposed Default Terminal Execution Specification.
 
+- [AMDGPU](https://www.amd.com/en/support/linux-drivers), AMD's open source driver for Radeon cards, is now available to be configured as `hardware.amdgpu` option.
```

Comment:
Wording nitpicks: `amdgpu` is not really _AMD's_ driver, it's the community driver, as opposed to AMD's internal AMDGPU-PRO stack. Also, the wording is awkward here, and almost none of the options actually configure `amdgpu` itself. Maybe just something like "Convenience options for AMD GPU drivers have been added under `hardware.amdgpu`"?